src/sections/CareersPerks/index.css

```diff
@@ -0,0 +1,101 @@
+@import "../../styles/all.css";
+
+.root {
+  position: relative;
+
+  padding-top: calc($spacing-84 + 40px);
+}
+
+.background {
+  z-index: 0;
+
+  width: calc(100% + 2 * $column-spacing);
+
+  margin-left: calc(-1 * $column-spacing);
+
+  svg {
+    width: 100%;
+  }
+
+  @include Breakpoint-mobileOnly {
+    display: none;
+  }
+}
+
+.background-mobile {
+  width: calc(100% + $gutter + $gutter);
+  height: 685px;
+
+  margin-top: $spacing-84;
+
+  margin-left: calc(-1 * $gutter);
+
+  svg {
+    width: 100%;
+  }
+
+  @include Breakpoint-tabletAndAbove {
+    display: none;
+  }
+}
+
+.content {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 1;
+
+  display: flex;
+  width: 100%;
+  max-width: $content-width-desktop;
+  margin-top: $spacing-224;
+
+  @include Breakpoint-tabletOnly {
+    width: 100vw;
```

Comment:
is it `vw` or `%` here?

Reply:
The result with `vw` or `100%` will be the same.